File: ManagemateAPI/ManagemateAPI/Management/M_Client/Input_Objects/Add_Client_Data.cs
using ManagemateAPI.Management.M_Session.Input_Objects;

namespace ManagemateAPI.Management.M_Client.Input_Objects
{
    public class Add_Client_Data
    {
        public Session_Data session { get; set; }
        public string surname { get; set; }
        public string name { get; set; }
        public string company_name { get; set; }
        public string nip { get; set; }
        public string phone_number { get; set; }
        public string email { get; set; }
        public string address { get; set; }
        public long city_id_fk { get; set; }
        public string postal_code { get; set; }
        public string comment { get; set; }
    }
}

File: Managemate/ManagemateAngular/src/app/shared/interfaces/API_Input_Models/Client_Models/Add_Client_Data.ts
import {Session_Data} from "../../API_Other_Models/Session_Models/SessionData";

export interface Add_Client_Data{
    session?: Session_Data;

    surname:string;
    name:string;
    company_name:string;
    nip: string;
    phone_number: string;
    email: string;
    address: string;
    city_id_fk:number;
    postal_code: string;
    comment: string;
}